src/corporacreator/corpora.py

```diff
@@ -42,7 +42,12 @@ def create(self):
         corpora_data[["sentence", "up_votes", "down_votes"]] = corpora_data[
             ["sentence", "up_votes", "down_votes"]
         ].swifter.apply(func=lambda arg: common_wrapper(*arg), axis=1)
-        for locale in corpora_data.locale.unique():
+        if self.args.langs:
+            locales = self.args.langs
+        else:
+            locales = corpora_data.locale.unique()
+            
+        for locale in locales:
```

Comment:
You should check that `self.args.langs` is a subset of `corpora_data.locale.unique()` and print an error message if it's not and also stop the program.

Reply:
@kdavis-mozilla -- requested changes made to README and catch if entered languages are actually subset of possible languages... 

also learned that Python has a `x.issubset(y)` function:)